code/game/machinery/vending.dm

```diff
@@ -661,7 +661,6 @@
 					/obj/item/reagent_containers/food/drinks/bottle/champagne/jericho = 1,
 					/obj/item/reagent_containers/food/drinks/bottle/champagne = 3,
 					/obj/item/reagent_containers/food/drinks/bottle/coconutmilk = 5,
-					/obj/item/reagent_containers/food/drinks/bottle/unathijuice = 2,
```

Comment:
Why are we removing this? its only been a recent addition

Reply:
It's a merge artifact issue. They resolved the merge improperly.